File: Practice/week28/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = -1;
    while (height > 8 || height < 1)
    {
        height = get_int("Height: ");
    }

    for (int i = 0, j = height - 1; i < height; i++, j--)
    {
        for (int k = 0; k < 2; k++)
        {
            if (k == 0)
            {
                if (j != 0)
                {
                    for (int l = 0; l < j; l++)
                    {
                        printf(" ");
                    }
                }
                for (int m = 0; m <= i; m++)
                {
                    printf("#");
                }
                printf("  ");
            }
            else
            {
                for (int m = 0; m <= i; m++)
                {
                    printf("#");
                }
            }
        }
        printf("\n");
    }
}

File: ps6/sentimental-mario-more/mario.py
# TODO

from cs50 import get_int

height = get_int("Height: ")

while height <= 0 or height >= 9:
    height = get_int("Height: ")

for i in range(height):
    for k in range(2):
        if k == 0:
            if height != 1:
                print(" " * (height - i - 1), end="")
            print("#" * (i + 1), end="")
            print("  ", end="")
        else:
            print("#" * (i + 1))